index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"/>
+    <title>Tweet-Twoo</title>
+</head>
+<body>
+    <div class="grid">
+        <section class="left-bar g">
+            <div class="logo g">
+                <img src="./twitter-svg/twitter-color-icon.svg" alt="Twitter logo">
+            </div>
+            <nav class="logo g">
+                <button class="effect"><img src="./twitter-svg/bird-house-icon.svg" alt="Home"> Home</button>
+                <button class="effect"><img src="./twitter-svg/hashtag-sign-icon.svg" alt="Explore"> Explore</button>
+                <button class="effect"><img src="./twitter-svg/bell-line-icon.svg" alt="Notifications"> Notifications</button>
+                <button class="effect"><img src="./twitter-svg/envelope-line-icon.svg" alt="Messages"> Messages</button>
+                <button class="effect"><img src="./twitter-svg/saved-icon.svg" alt="Bookmarks"> Bookmarks</button>
+                <button class="effect"><img src="./twitter-svg/checklist-icon.svg" alt="Lists"> Lists</button>
+                <button class="effect"><img src="./twitter-svg/user-profile-icon.svg" alt="Profile"> Profile</button>
+                <button class="effect"><img src="./twitter-svg/more-options-ellipsis-icon.svg" alt="More"> More</button>
+            </nav>
+            <div class="tweet g"><button>Tweet</button></div>
```

Comment:
I think it would be a good idea to add more comments within your code for readability/navigating which would help explain what sections are doing

Reply:
100% i noticed that while reading other peoples code haha, i forgot others will need to read mine